imports/plugins/core/cart/server/no-meteor/util/updateCartItemsForVariantChanges.js

```diff
@@ -5,15 +5,27 @@
  *   the cart items were last updated. The return object will have a `didUpdate`
  *   boolean property that you can check to see whether any changes were made.
  * @param {Object[]} items Cart items
- * @param {String} variantId ID of variant to update items for
+ * @param {Object} catalogProductVariant The updated variant
  * @param {Object} prices Various updated price info for this variant
  * @returns {Object} { didUpdate, updatedItems }
  */
-export default function updateCartItemsForVariantPriceChange(items, variantId, prices) {
+export default function updateCartItemsForVariantChanges(items, catalogProductVariant, prices) {
   let didUpdate = false;
 
   const updatedItems = items.map((item) => {
-    if (item.variantId !== variantId) return item;
+    if (item.variantId !== catalogProductVariant.variantId) return item;
```

Comment:
Should we also be checking for stuff like weight which would affect shipping calculations?